src/mlpack/methods/ann/layer/layer_types.hpp

```diff
@@ -33,7 +33,6 @@
 #include <mlpack/methods/ann/layer/parametric_relu.hpp>
 #include <mlpack/methods/ann/layer/reinforce_normal.hpp>
 #include <mlpack/methods/ann/layer/select.hpp>
-#include <mlpack/methods/ann/layer/cross_entropy_error.hpp>
```

Comment:
Not sure what happened here, but we should not remove the recently introduced cross entropy error function.